webapp/routes/auth.js

```diff
@@ -0,0 +1,106 @@
+import express from "express"
+import passport from "passport"
+import LocalStrategy from "passport-local"
+import TwitterStrategy from "passport-twitter"
+import FacebookStrategy from "passport-facebook"
+import GoogleStrategy from "passport-google-oauth20"
+import { users } from "./../models/storage"
+import secretStorage, {auth as authStorage} from "./../models/secretStorage"
+import jwt from "jsonwebtoken"
+
+const router = express.Router();
+
+function checkAuthentication(idProperty, idPropertyValue, password, done) {
+  let auth = authStorage.find((item) => item[idProperty] === idPropertyValue);
+
+  var user, authInfo;
```

Comment:
please don't use `var` declaration name due to not being block-scoped and out-of-date practice